installer/install.sh

```diff
@@ -83,15 +83,33 @@ fi
 # get running machine from conf file
 if [ -r /etc/machine.conf ]; then
     read_conf_file "/etc/machine.conf"
+    machine_conf_path=/etc/machine.conf
 elif [ -r /host/machine.conf ]; then
     read_conf_file "/host/machine.conf"
+    machine_conf_path=/host/machine.conf
 elif [ "$install_env" != "build" ]; then
     echo "cannot find machine.conf"
     exit 1
 fi
 
+onie_running_arch=`cat $machine_conf_path | grep onie_arch | sed -n "s/^onie_arch=\(.*\)$/\1/p"`
+onie_running_switch_asic=`cat $machine_conf_path | grep onie_switch_asic | sed -n "s/^onie_switch_asic=\(.*\)$/\1/p"`
```

Comment:
>onie_running_switch_asic [](http://example.com/codeflow?start=0&length=24)

I am not sure if the code is working for old version of ONIE. Suggest you test the scenarios that these variables are not existing.